indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -162,7 +161,21 @@ def _process(self, action, comment):
             argmap['files'] = EditingFilesField(self.event, self.contrib, self.editable_type, allow_claimed_files=True,
                                                 required=True)
         args = parser.parse(argmap)
-        review_editable_revision(self.revision, session.user, action, comment, args['tags'], args.get('files'))
+        service_url = editing_settings.get(self.event, 'service_url')
+
+        new_revision = review_editable_revision(self.revision, session.user, action, comment, args['tags'],
+                                                args.get('files'))
+
+        publish = True
+        if service_url:
+            try:
+                resp = service_handle_review_editable(self.editable, action, self.revision, new_revision)
+                publish = resp.get('publish', True)
+            except ServiceRequestFailed:
+                raise ServiceUnavailable(_('Failed processing review, please try again later.'))
+
+        if publish and action in (EditingReviewAction.accept, EditingReviewAction.update_accept):
+            publish_editable_revision(new_revision or self.revision)
         return '', 204
 
```

Comment:
When the submitter creates a new revision after "request changes", we currently do not notify the service. `RHCreateSubmitterRevision` needs some service-reloated logic as well.

Reply:
Thought about the same, but had the impression we didn't actually need it because it wasn't considered as a "revision". Nonetheless, sounds good to me.